Meadowlark-js/docker/docker-compose-opensearch.yml

```diff
@@ -0,0 +1,48 @@
+version: '3'
```

Comment:
I almost asked for this separation in the 3-node cluster ticket. Thank you for doing that.